packages/cardano-services/test/Asset/TypeormAssetProvider.test.ts

```diff
@@ -0,0 +1,171 @@
+import { Cardano, ProviderError } from '@cardano-sdk/core';
+import {
+  TypeOrmNftMetadataService,
+  TypeormAssetProvider,
+  createDnsResolver,
+  getConnectionConfig,
+  getEntities
+} from '../../src';
+import { logger } from '@cardano-sdk/util-dev';
+import { sleep } from '../util';
+
+const testHandleAssetId = Cardano.AssetId(
```

Comment:
Which is the source of these data?
If source are fixtures, these data should be read from DB, not hardcoded in the test, otherwise when we re-generate the fixtures the test will fail.

Reply:
I read these data from [here](https://github.com/input-output-hk/cardano-js-sdk/blob/cabc31e22b738ef0d9fccd39d591fa41853c6f5f/packages/cardano-services/test/jest-setup/snapshots/handle.sql#L257-L264)

Reply:
@greatertomi please refer to fixtures setup in TypeormStakePoolProvider.

Reply:
Fixed on 82b3857